scripts/test/Muon/loading_tab/loadwidget_presenter_test.py

```diff
@@ -71,6 +70,12 @@ def setUp(self):
         self.addCleanup(self.popup_patcher.stop)
         self.popup_mock = self.popup_patcher.start()
 
+        def setGroupAndPairsToEmptyList(grouping_context):
+            grouping_context._groups = []
+            grouping_context._pairs = []
+        self.group_context.reset_group_and_pairs_to_default = mock.MagicMock(
```

Comment:
incorrect indent

Reply:
Indentation is correct, it works as intended.